Source/Diagnostics/WarpXIO.cpp

```diff
@@ -122,49 +122,49 @@ WarpX::InitFromCheckpoint ()
         {
             std::istringstream lis(line);
             lis.exceptions(std::ios_base::failbit | std::ios_base::badbit);
-            for (int i = 0; i < istep.size(); ++i) {
+            for (auto& ist : istep) {
                 lis >> word;
-                istep.at(i) = std::stoi(word);
+                ist = std::stoi(word);
```

Comment:
Could we call this `istep_lev` (I think that's what it is)?

Reply:
good idea!